test/TestUtils.py

```diff
@@ -185,3 +187,54 @@ def test_expand_paths_vars(monkeypatch):
     monkeypatch.setenv('TEST_PATH', test_path)
     assert utils.expand_paths_vars(['~']) == [os.path.expanduser('~')]
     assert utils.expand_paths_vars(['$TEST_PATH']) == [test_path]
+
+
+def test_get_yaml_files_no_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.setenv('PATH', '')
+    utils.get_yaml_files(options)
+    msg1, msg2 = capsys.readouterr().out.splitlines()[:2]
+    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert msg2.startswith('Warning: Failed to locate command:')
+
+
+def test_get_yaml_files_not_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.chdir('/tmp')
+    utils.get_yaml_files(options)
+    msg1, msg2 = capsys.readouterr().out.splitlines()[:2]
+    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert msg2.startswith('Warning: Failed to discover yaml files to lint using git:')
+
+
+@pytest.mark.parametrize(
+    'is_in_git',
+    [
+        (True),
+        (False)
+    ]
+)
```

Comment:
Just `(value)` is that same as `value` so braces are unnecessary. But if you do `(value, )` it's a tuple. Still, it's not needed there because you have just one param.

This linter error https://github.com/ansible/ansible-lint/pull/738/checks?check_run_id=611810102#step:10:26 is about having that outer list converted into a tuple. Not the params.

Also, the params list is short enough so it could fit in one line (unless you want to multiline it intentionally):
```suggestion
@pytest.mark.parametrize('is_in_git', (True, False))
```